src/screens/authentication/UsernameRequest/styles.js

```diff
@@ -6,57 +6,60 @@ export const PLACEHOLDER_TEXT_COLOR = theme.colors.gray
 export const ICON_WIDTH = moderateScale(66.53)
 export const ICON_HEIGHT = moderateScale(51.41)
 
-const ICON_TOP_DISPLACEMENT_IN_DP = 89
-
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
-    top: verticalScale(175)
-  },
-  iconBackground: {
-    alignSelf: 'center',
-    position: 'absolute',
-    top: verticalScale(ICON_TOP_DISPLACEMENT_IN_DP + 10),
-    width: moderateScale(62),
-    height: moderateScale(42),
-    backgroundColor: 'rgba(94, 206, 132, 0.2)'
+    display: 'flex',
+    alignItems: 'center'
   },
   icon: {
-    alignSelf: 'center',
-    position: 'absolute',
-    top: verticalScale(ICON_TOP_DISPLACEMENT_IN_DP)
+    marginTop: verticalScale(105)
   },
   title: {
-    alignSelf: 'center',
+    marginTop: verticalScale(33),
     marginBottom: verticalScale(12)
   },
   description: {
-    alignSelf: 'center',
     textAlign: 'center',
     width: theme.sizes.containerWidth
   },
   textInput: {
     width: theme.sizes.containerWidth,
     height: theme.sizes.base,
     marginTop: verticalScale(35),
-    marginBottom: verticalScale(30),
+    marginBottom: verticalScale(5),
     padding: theme.sizes.padding,
     paddingLeft: scale(14),
-    alignSelf: 'center',
     borderRadius: theme.sizes.border,
     color: theme.colors.black,
     textDecorationLine: 'none',
     backgroundColor: theme.colors.white,
     ...theme.fonts.body,
     ...theme.shadows
   },
+  submitButton: {
+    marginTop: verticalScale(30)
+  },
   error: {
-    alignSelf: 'center',
-    marginBottom: theme.sizes.margin
+    marginLeft: scale(24),
+    alignSelf: 'flex-start',
+    marginBottom: theme.sizes.margin,
+    color: theme.colors.error,
+    fontSize: theme.sizes.error
   },
   submitIndicator: {
-    alignSelf: 'center',
     marginBottom: theme.sizes.margin
+  },
+  // Sugestiones
+  suggestionContainer: {
+    borderColor: '#DADADA',
+    borderRadius: scale(5),
+    borderWidth: scale(1)
+  },
+  lastSuggestionBox: {
+    borderBottomWidth: scale(0)
+  },
+  suggestionTitle: {
+    ...theme.fonts.h3
```

Comment:
I don't see this style being applied, consider removing it as it could be unused code.